openfisca_core/decompositions.py

```diff
@@ -31,7 +32,7 @@ def new_test_case_array(holder, array):
                     exc.simulation_index = simulation_index
                     raise
                 holder = simulation.get_holder(node['code'])
-                column = holder.column
+                column = make_column_from_variable(holder.variable)
```

Comment:
Indicated it in `columns.py`.

This part of the code is full of Byriani, not properly documented, and we're not sure about what it does exactly. From there, two strategies are possible when we want to publish a change that impact this code:

- We try to understand how this piece of code work. When we succeed, we adapt the code so that it can work with the a Variable instead of a Column.
    - Advantage: cleaner once its one, _if_ we don't pollute the new `Variable` class with stuff from `Column` that we don't really want to keep.
    - Drawback: high risk of getting lost, and to spend more time migrating this module we don't even know than making the changes we want to do.
- We consider this piece of code as legacy that we are maintaining in a minimalistic way. We try to avoid modifying it. If it needs a column, we give it a column, hoping that one day it will be rebuilt or remove.
    - Advantage: Our time investment is more focused
    - Drawback: Some legacy modules stay in the code

I chose strategy 2, as columns were needed in several part of the code I don't want to deal with.